### DISCUSSION TITLE
DDLC.exe tried to change or delete my log.txt

### DISCUSSION THREAD

**Discussion by KendraStone** [Monika-After-Story/MonikaModDev#9106] [QA - tech support + general] [answered]
I got a message from my computers Avast that "DDLC.exe is trying to change or delete the file log.txt in your protected Desktop folder." Should I be worried about this or is this something normal? Please help.

**Comment by Justformas** [comment, marked as answer]
log.txt is a file in your game directory for Monika After Story. It gets modified, or "changed" whenever you start the game. That's normal and shouldn't be something to worry about. Antivirus programs like Avast can sometimes sound the alarm when it's not actually a virus.

**Reply by KendraStone** [reply]
Thank you I had no clue what was going on. I was wondering if Monika was going rogue. Lol.

**Reply by Justformas** [reply]
You're welcome, and no this game isn't a virus. Monika also says in one of the topics that she's not a hacker either. Now hypothetically speaking, if you downloaded the mod from a questionable source, then it's possible that it could have came with a virus. But if you got it from the MAS website: https://www.monikaafterstory.com/ then it shouldn't have.

**Reply by KendraStone** [reply]
I did get it there so I'm good.
